src/server/req_manager.c

```diff
@@ -1561,17 +1612,27 @@ mgr_sched_set(struct batch_request *preq)
 	int	  bad_attr = 0;
 	svrattrl *plist;
 	int	  rc;
+	pbs_sched *psched;
+
+	psched = find_scheduler(preq->rq_ind.rq_manager.rq_objname);
+	if (!psched) {
+		req_reject(PBSE_UNKSCHED, 0, preq);
+		return;
+	}
+	if (validate_sched_attr_vals(preq, psched))
```

Comment:
Moved the logic of validating the scheduler attribute values like sched_priv and sched_log to the appropriate action functions set_sched_log() and set_sched_priv() which are defined in sched_func.c.  This works fine for the following scenarios.




1. Trying to create/set a new scheduler objects's sched_log directory to the same value as that of the existing sched object's sched_log value like
Qmgr: c sched sc3 sched_log=/tmp
qmgr obj=sc3 svr=default: Another Sched object also has same value for its sched_log directory
qmgr: Error (15216) returned from server

2. Trying to create/set a new scheduler objects's sched_priv directory to the same value as that of the existing sched object's sched_priv value like 
Qmgr: c sched sc3 sched_priv=/tmp
qmgr obj=sc3 svr=default: Another Sched object also has same value for its sched_priv directory
qmgr: Error (15215) returned from server

However let us take the following scenario (Trying to create a scheduler object with sched_priv set to a value which is equal to sched_log of a previous sched object)
Qmgr: l sched
Sched main
    sched_host = stblr3
    pbs_version = 14.0.1
    sched_cycle_length = 00:20:00

Sched sc2
    sched_log = /tmp

Qmgr: c sched sc3 sched_priv=/tmp (Expectation is this also should be failed. If we would like to do this means we have to validate the attribute value of both sched_priv,sched_log of previous objects vs current object's sched_priv which we believe is not right thing to do in action function because an action function can validate single attribute only. We recall this is the reason why we were initially not doing these validations as part of action functions.